src/main/java/com/alexx/employeesAndDepartments/services/DepartmentService.java

```diff
@@ -73,12 +77,14 @@ public DepartmentDTO getDepartment(Integer id) {
         DepartmentEntity departmentEntity = departmentRepository.findById(id).get();
         DepartmentDTO departmentDTO = modelMapper.map(departmentEntity, DepartmentDTO.class);
         departmentDTO.setListOfEmployees(employeeListOfDepartment(id));
+        log.info("Successfully");
```

Comment:
Please read about best practices in logging and correct your messages

Reply:
ok, thank you